gsaizonou@gsaizonou-HP-EliteBook-840-G1:~/Documents/CBT/CBTWebGestion$ npm run dev

> rest-express@1.0.0 dev
> NODE_ENV=development tsx server/index.ts

node:internal/deps/undici/undici:13510
      Error.captureStackTrace(err);
            ^

TypeError: fetch failed
    at node:internal/deps/undici/undici:13510:13
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async performDiscovery (/home/gsaizonou/Documents/CBT/CBTWebGestion/node_modules/openid-client/src/index.ts:1359:14)
    at async Module.discovery (/home/gsaizonou/Documents/CBT/CBTWebGestion/node_modules/openid-client/src/index.ts:1280:14)
    at async memoize.maxAge (/home/gsaizonou/Documents/CBT/CBTWebGestion/server/replitAuth.ts:17:12)
    at async setupAuth (/home/gsaizonou/Documents/CBT/CBTWebGestion/server/replitAuth.ts:75:18)
    at async registerRoutes (/home/gsaizonou/Documents/CBT/CBTWebGestion/server/routes.ts:17:3)
    at async <anonymous> (/home/gsaizonou/Documents/CBT/CBTWebGestion/server/index.ts:40:18) {
  [cause]: AggregateError [ETIMEDOUT]: 
      at internalConnectMultiple (node:net:1122:18)
      at afterConnectMultiple (node:net:1689:7) {
    code: 'ETIMEDOUT',
    [errors]: [
      Error: connect ETIMEDOUT 172.64.152.210:443
          at createConnectionError (node:net:1652:14)
          at Timeout.internalConnectMultipleTimeout (node:net:1711:38)
          at listOnTimeout (node:internal/timers:583:11)
          at process.processTimers (node:internal/timers:519:7) {
        errno: -110,
        code: 'ETIMEDOUT',
        syscall: 'connect',
        address: '172.64.152.210',
        port: 443
      },
      Error: connect ETIMEDOUT 2606:4700:4400::ac40:98d2:443
          at createConnectionError (node:net:1652:14)
          at Timeout.internalConnectMultipleTimeout (node:net:1711:38)
          at listOnTimeout (node:internal/timers:583:11)
          at process.processTimers (node:internal/timers:519:7) {
        errno: -110,
        code: 'ETIMEDOUT',
        syscall: 'connect',
        address: '2606:4700:4400::ac40:98d2',
        port: 443
      },
      Error: connect ETIMEDOUT 104.18.35.46:443
          at createConnectionError (node:net:1652:14)
          at Timeout.internalConnectMultipleTimeout (node:net:1711:38)
          at listOnTimeout (node:internal/timers:583:11)
          at process.processTimers (node:internal/timers:519:7) {
        errno: -110,
        code: 'ETIMEDOUT',
        syscall: 'connect',
        address: '104.18.35.46',
        port: 443
      },
      Error: connect ENETUNREACH 2606:4700:4400::6812:232e:443
          at createConnectionError (node:net:1652:14)
          at afterConnectMultiple (node:net:1682:16) {
        errno: -101,
        code: 'ENETUNREACH',
        syscall: 'connect',
        address: '2606:4700:4400::6812:232e',
        port: 443
      }
    ]
  }
}

Node.js v20.19.4